swift/apple/FirezoneKit/Sources/FirezoneKit/Views/MenuBar.swift

```diff
@@ -484,6 +481,19 @@ public final class MenuBar: NSObject, ObservableObject {
     resourceAddressItem.target = self
     subMenu.addItem(resourceAddressItem)
 
+    // Resource toggle
+    if resource.canToggle {
+      subMenu.addItem(NSMenuItem.separator())
+      enableToggle.action = #selector(resourceToggle(_:))
+      enableToggle.title = "Enabled"
+      enableToggle.toolTip = "Toggle resource"
```

Comment:
```suggestion
      enableToggle.title = "Disable"
      enableToggle.toolTip = "Enable or Disable this Resource"
```

The `title` here should be a verb to indicate it is an actionable item. `Enable this Resource` if state is disabled and `Disable this Resource` if state is enabled.